text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+A significant percentage of these have not been active in the recent past.
+
+We have kept PRs and issues open so people could more easily find the discussions,
+but this has instead given a negative feeling of staleness, as RFCs linger open without new feedback.
+
+## Detailed design
+
+To address the issues raised in the “Motivation” section, I will propose changes to the current process.
+
+### Implement Core Champion
+
+To make sure that RFCs receive adequate support from the team, Ember CLI has implemented the idea of a champion associated with each RFC.
+The goal is that in seeking a champion from the team,
+the RFC author starts a dialogue with the team and gets some early feedback.
+The champion is then responsible for representing the RFC in team meetings, and for shepherding its progress.
+
+This will be imported into emberjs/rfcs.
+
+### Improve the lifecycle of RFCs
+
+To address the triaging and inactivity problem of RFCs, I introduce the concept of FCP to close.
+
+Closing an RFC should be viewed as another triaging tool, not as a rejection of the RFC.
+Sometimes a rewrite of an RFC would be so fundamental that it would benefit of a fresh discussion in a new thread.
+Sometimes the original author is no longer active (Champions should help here as well),
+and someone else might want to take over the work in a new RFC.
+Sometimes the timing might not be right, or the feature might have been addressed some other way, and yes,
+sometimes it might be something that is not aligned with the team's values for the project.
+
+A good example of this is the [Named Blocks RFC](https://github.com/emberjs/rfcs/blob/master/text/0226-named-blocks.md#motivation),
+which lists in the motivation section previous attempts at similar ideas.
+
+Like the FCP to merge process, once an RFC is marked as FCP to close there will be a period of one week where people can raise new concerns.
+After that period of one week, the respective team will review and close the RFC or extend the period for another week.
+
+### Merge ember-cli/rfcs into emberjs/rfcs
+
+Part of removing confusion about the process is having a central repository for all Ember RFCs.
+
+To achieve merging ember-cli/rfcs into emberjs/rfcs the following will be done:
+
+* Remove active/completed concepts from ember-cli/rfcs
+  * Add front matter
+  * Copy both active and completed RFC files into `text` of emberjs/rfcs
+* Transfer active PRs and Issues to emberjs/rfcs
+* Archive ember-cli/rfcs
+
+There are some concerns about links breaking when we move the files to emberjs/rfcs,
+but given the fact that ember-cli/rfcs had the concept of active/completed by moving the files into different folders,
+links were already being broken.
+
```

Comment:
Phrase more positively - It is expected that some links will break in the process of moving files to emberjs/rfcs. This does not introduce any new concerns, since ember-cli/rfcs already had a process to move files into active/completed folders, which also breaks the links.